server/src/client/client.service.ts

```diff
@@ -99,6 +102,14 @@ export class ClientService {
     } else {
       this.gateway.server.in(target).socketsJoin(resource.id);
 
+      const fieldList = Object.keys(
+        resource.dtoField ? (resource.dtoField as {}) : dto,
+      );
+
+      const options: PermittedFieldsOptions<AppAbility> = {
```

Comment:
You might have made a mistake with resolving merge conflicts as this code is old and unusable